db/migrations/0003_alter_moviesession_cinema_hall_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-03-04 18:52
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_cinemahall_movie_moviesession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='cinema_halls', to='db.cinemahall'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='movie',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movies', to='db.movie'),
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app per PR in Django projects since it helps to avoid `migrations` folder overload in the future. Represent all changes done to models in one migration file per app per PR, please